src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2723,13 +2723,9 @@ private static void CheckInterfaceMember(Symbol member, DiagnosticBag diagnostic
                         {
                             diagnostics.Add(ErrorCode.ERR_EnumsCantContainDefaultConstructor, m.Locations[0]);
                         }
-                        else if (m.DeclaredAccessibility != Accessibility.Public)
-                        {
-                            diagnostics.Add(ErrorCode.ERR_ParameterlessStructCtorsMustBePublic, m.Locations[0]);
```

Comment:
Remove ERR_ParameterlessStructCtorsMustBePublic enum member and corresponding string from resources?